tests/sqlparser_bigquery.rs

```diff
@@ -2566,3 +2566,24 @@ fn test_struct_trailing_and_nested_bracket() {
         )
     );
 }
+
+#[test]
+fn test_export() {
```

Comment:
Since this introduces a new statement, can we include an assertion on the AST for one of the statements, checking that the returned AST looks like what we expect? [For example](https://github.com/apache/datafusion-sqlparser-rs/blob/1a7e8d969ae509d99f5f9aa1136092da002a77c6/tests/sqlparser_bigquery.rs#L2203-L2232)


Also could we add coverage for some scenarios like the following?
```
"EXPORT DATA OPTIONS()";
"EXPORT DATA AS SELECT ...";
"EXPORT DATA WITH CONNECTION ... AS SELECT ..."
```